benchmarker/frameworks/do_mkl.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from benchmarker.util.abstractprocess import Process
+
+from .i_gemm import IGEMM
+
+
+class Benchmark(IGEMM):
+    def __init__(self, params, remaining_args=None):
+        super().__init__(params, remaining_args)
+        assert self.params["problem"]["precision"] in ["FP32", "mixed"]
+
+    def run(self):
+        if "nb_gpus" in self.params:
+            if self.params["nb_gpus"] > 0:
+                raise Exception("cblas does not work on GPU")
+        # TODO(Alex): this does not work inless the binaries are copied to site_packages
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        path_binary = os.path.join(dirname, "../kernels/gemm/mkl/main")
```

Comment:
This path seems to be wrong... actually this might be a bigger problem...

Reply:
Try this (or something similar) and check that it works

```python
path_binary = os.path.join(dirname, "../kernels", self.params["problem"], self.params["framework"], "main")
```

Reply:
Thank you, but `self.params["problem"] `is a `dict`, so I added get function after it.
`path_binary = os.path.join(dirname, "../kernels", self.params["problem"].get("name"), self.params["framework"], "main")`